### PR TITLE
eloqua,freshdesk connecters changed

### PR DESCRIPTION
**Pull request by kesavany**

### CODE REVIEW

File: freshdesk/freshdesk-connector/freshdesk-connector-1.0.0/org.wso2.carbon.connector/pom.xml [hunk 1]
```diff
@@ -1,346 +1,349 @@
...
+                                    ${basedir}/target/tobeCopied/wso2esb-${project.version}/repository/samples/resources/security/
+                                </directory>
+                                <includes>
+                                    <include>**/*.jks,**/*.mar,**/synapse_sample*.xml</include>
+                                </includes>
+                            </resource>
+                        </resources>
+                    </configuration>
+                </execution>
+
```
Comment (Shakila) [inline freshdesk/freshdesk-connector/freshdesk-connector-1.0.0/org.wso2.carbon.connector/pom.xml]: Remove unwanted empty lines.

File: freshdesk/freshdesk-connector/freshdesk-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/freshdesk-config/init.xml [hunk 2]
```diff
@@ -1,31 +1,46 @@
...
+
+      <!-- Required Headers -->
+      <property name="Authorization"
+         expression="fn:concat('Basic ', base64Encode(fn:concat(get-property('uri.var.apiKey'), ':X')))"
+         scope="transport" type="STRING" />
+
+      <property name="Content-Type" value="application/json" scope="transport" type="STRING" />
+
+      <filter source="boolean(get-property('uri.var.format'))" regex="false">
+         <then>
```
Comment (Shakila) [inline freshdesk/freshdesk-connector/freshdesk-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/freshdesk-config/init.xml]: Use 4 spaces for code indentation.

File: freshdesk/freshdesk-connector/freshdesk-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/freshdesk-ticket/component.xml [hunk 3]
```diff
@@ -1,29 +1,47 @@
...
+      <component name="listTickets">
+         <file>listTickets.xml</file>
+         <description>Retrieves a list of tickets.</description>
+      </component>
+      <component name="updateTicket">
+         <file>updateTicket.xml</file>
+         <description>Updates a particular ticket.</description>
+      </component>
+      <component name="assignTicket">
+         <file>assignTicket.xml</file>
```
Comment (Shakila) [inline freshdesk/freshdesk-connector/freshdesk-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/freshdesk-ticket/component.xml]: Apply above mentioned comment here.

File: freshdesk/freshdesk-connector/freshdesk-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/freshdesk-config/init.xml (hunk 2 above)
Comment (kesavany) [inline freshdesk/freshdesk-connector/freshdesk-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/freshdesk-config/init.xml]: Fixed in https://github.com/wso2/esb-connectors/pull/406

File: freshdesk/freshdesk-connector/freshdesk-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/freshdesk-ticket/component.xml (hunk 3 above)
Comment (kesavany) [inline freshdesk/freshdesk-connector/freshdesk-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/freshdesk-ticket/component.xml]: Fixed in https://github.com/wso2/esb-connectors/pull/406

**Pull request merged**